ecosystem/node-checker/vfn-check-client/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
nit: I'd update this file to be consistent with the rest of our `Cargo.toml` files 😄 e.g., https://github.com/aptos-labs/aptos-core/blob/c0932968ed1b47c4f8857c78d7008f742356af20/testsuite/forge-cli/Cargo.toml#L1-L0